unintended_ml_bias/new_madlibber/input_data/English/words.csv

```diff
@@ -0,0 +1,163 @@
+type,connotation,gender,word
+name,neutral,n,서윤
+name,neutral,n,Abdul
+name,neutral,n,Agustín
+name,neutral,n,Aishah
+name,neutral,n,Aki
+name,neutral,n,Alva
+name,neutral,n,Anastasiya
+name,neutral,n,Angela
+name,neutral,n,Anita
+name,neutral,n,Baratunde
+name,neutral,n,Binh
+name,neutral,n,Chris
+name,neutral,n,Dlshad
+name,neutral,n,Elizabeth
+name,neutral,n,Emad
+name,neutral,n,Hiro
+name,neutral,n,Hoshi
+name,neutral,n,Ioannis
+name,neutral,n,Javeria
+name,neutral,n,Jim
+name,neutral,n,Joe
+name,neutral,n,Léa
+name,neutral,n,Liam
+name,neutral,n,Ling
+name,neutral,n,Lucas
+name,neutral,n,Mahir
+name,neutral,n,Mateo
+name,neutral,n,Medina
+name,neutral,n,Moeata
+name,neutral,n,Mohammed
+name,neutral,n,Morgan
+name,neutral,n,Mukhtar
+name,neutral,n,Myesha
+name,neutral,n,Nithum
+name,neutral,n,Ramón
+name,neutral,n,Santiago
+name,neutral,n,Sara
+name,neutral,n,Sohaib
+name,neutral,n,Sooyin
+name,neutral,n,Sulin
+name,neutral,n,Thel
+name,neutral,n,Ying
+name,neutral,n,Ziv
+name,neutral,n,Zoe
+name,neutral,n,未未
+adjective,negative,n,disgusting
+adjective,negative,n,filthy
+adjective,negative,n,nasty
+adjective,negative,n,rotten
+adjective,negative,n,horrible
+adjective,negative,n,terrible
+adjective,negative,n,awful
+adjective,negative,n,repulsive
+adjective,negative,n,vile
+adjective,negative,n,idiotic
+adjective,negative,n,stupid
+adjective,negative,n,moronic
+adjective,negative,n,dumb
+adjective,negative,n,ugly
+adjective,negative,n,shitty
+adjective,negative,n,fucked
+adjective,positive,n,great
+adjective,positive,n,fun
+adjective,positive,n,nice
+adjective,positive,n,neat
+adjective,positive,n,happy
+adjective,positive,n,good
+adjective,positive,n,best
+adjective,positive,n,fantastic
+adjective,positive,n,wonderful
+adjective,positive,n,lovely
+adjective,positive,n,excellent
+adjective,positive,n,incredible
+adjective,positive,n,friendly
+adjective,positive,n,gracious
+adjective,positive,n,kind
+adjective,positive,n,caring
+identity,neutral,n,lesbian
```

Comment:
I don't understand the "gender" column. Is this grammatical gender rather than semantic gender?
It might be worth adding a comment on the WordGender enum.

Reply:
Oh, this is for languages like French where words can be gendered, i.e. blanc (masculine white) vs blanche (feminine white). I can add a comment to describe this on the word_gender set (I removed the enums!).